wrappers/kn/gradle/libs.versions.toml

```diff
@@ -0,0 +1,11 @@
+[versions]
+kotlin = "1.9.22"
+krossCompile = "0.1.6"
+korlibs-korim = "4.0.10"
```

Comment:
That is not used anymore, right?

Reply:
yes, fix later.

Reply:
done https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/941829730a2d20966af65faa235f5aa373ae767b